controls/togglebutton/getting-started.md

```diff
@@ -65,7 +61,7 @@ public static class MauiProgram
 }           
 ```
 
-> For a runnable example with the ToggleButton Getting Started scenario, see the [SDKBrowser Demo Application]({%slug sdkbrowser-app%}) and go to **Button > Getting Started** category.
+>tip For a runnable example with the ToggleButton Getting Started scenario, see the [SDKBrowser Demo Application]({%slug sdkbrowser-app%}) and go to the **ToggleButton > Getting Started** category.
```

Comment:
Either switch all similar callouts throughout the PR to tip, or return this one to the orange info style. :) Since you're in a hurry, switching back to a regular callout might be best here.